modules/cnv-monitoring-vm-migration-cli.adoc

```diff
@@ -15,7 +15,7 @@ of the `VirtualMachineInstance` configuration.
 ----
 $ oc describe vmi vmi-fedora
 ----
-+
+
```

Comment:
This `+` should still be here, but it also needs a `+` before the `oc describe` codeblock.

Reply:
Thanks @adellape. Done. Should be ready for merge and pick.